app/components/editor-document-title.hbs

```diff
@@ -5,7 +5,7 @@
 {{else}}
   {{#if this.active}}
     <div class="au-c-app-chrome__title-group {{if this.error "au-c-input--error"}}">
-      <Input class="au-c-app-chrome__title-input" placeholder="Naamloos document" @type="text" @value={{this.title}} {{on "input" this.setTitle}} @autofocus="autofocus"  />
```

Comment:
I removed the autofocus part since it seemed that it wasn't working. I think something like an `{{autofocus}}` modifier is needed.